src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9598,8 +9601,13 @@ private bool ScanParenthesizedImplicitlyTypedLambda(Precedence precedence)
                 && (!this.IsInQuery || !IsTokenQueryContextualKeyword(this.PeekToken(1)))
                 && this.PeekToken(2).Kind == SyntaxKind.CommaToken)
             {
+                if (!IsFeatureEnabled(MessageID.IDS_FeatureTuples))
+                {
+                    return true;
+                }
```

Comment:
Yes, the code for this `if` body was `return true;` before tuples came along. Will add comment.